### PR TITLE
Update v3.26 release notes

### PR DESCRIPTION
**Pull request by vikaspotluri123**

### CODE REVIEW

File: src/release-notes.md
Comment (joshcosta) [inline src/release-notes.md:30]: /n

File: src/release-notes.md
Comment (joshcosta) [inline src/release-notes.md:43]:
```suggestion
  - During transition periods, such as between the Fall and Winter semester, the older semester will be used for GPA calculations until it is archived.
```

File: src/release-notes.md
Comment (joshcosta) [inline src/release-notes.md:30]:
```suggestion
3. Account creation after trying to import a course will work now

 - Previously, you could get a cryptic error
```

File: src/release-notes.md
Comment (joshcosta) [inline src/release-notes.md:25]:
```suggestion
2. Logging in to a school's site on Gradebook will properly take you to the correct school
```

**Review by joshcosta — APPROVED** [review]
Approved after requested changes

File: src/release-notes.md
Comment (vikaspotluri123) [reply src/release-notes.md:25]: a `different` school on Gradebook will take you to `your` correct school?

Comment (vikaspotluri123) [reply src/release-notes.md:25]: Whatever you decide, please do a quick UI review before merging!

Comment (joshcosta) [reply src/release-notes.md:25]: I like your wording

**Pull request merged**
